Detectors/TPC/calibration/SpacePoints/src/TrackInterpolation.cxx

```diff
@@ -184,7 +184,7 @@ void TrackInterpolation::prepareInputTrackSample(const o2::globaltracking::RecoC
           continue;
         }
         mSeeds.push_back(mRecoCont->getITSTrack(gidTable[GTrackID::ITS]).getParamOut());
-        mSeeds.back().setPID(mRecoCont->getTrack(vidOrig).getPID());
+        mSeeds.back().setPID(mRecoCont->getTrack<o2::track::TrackParCov>(vidOrig).getPID());
```

Comment:
Sorry, actually it should be `mRecoCont->getTrackParam(vidOrig).getPID()` to cover all cases.